components/scream/src/share/io/output_manager.hpp

```diff
@@ -110,12 +110,12 @@ class OutputManager
   ekat::Comm                                   atm_comm;
   ekat::Comm                                   pio_comm; 
   ekat::ParameterList                          m_params;
-  std::shared_ptr<const FieldManager<Real>> m_device_field_manager;
+  std::shared_ptr<const FieldManager<Real>>    m_device_field_manager;
```

Comment:
MInor: there is no Device vs Host field manager. Each field contains both a host and a device view. The name 'device_field_manager' gives the false feeling that it has only device stuff, or that there is such a thing as a 'host_field_manager'. I would just call it `m_field_manager`.